Metalama.Framework.Engine/Templating/TemplateMemberSymbolClassifier.cs

```diff
@@ -27,6 +27,7 @@ public static bool IsDynamicParameter( ITypeSymbol? type )
             null => false,
             IDynamicTypeSymbol => true,
             IArrayTypeSymbol { ElementType: IDynamicTypeSymbol } => true,
+            INamedTypeSymbol { IsGenericType: true } genericType when genericType.TypeArguments.Any( IsDynamicParameter ) => true,
```

Comment:
Should the array case recursively call `IsDynamicParameter` as well, to catch e.g. `dynamic[][]`?